src/main/cpp/t08_queen.cpp

```diff
@@ -21,8 +21,24 @@
 #include <vector>
 
 using namespace std;
+int C[10];
+int QC1(int Q1, int Q2, int Q4) {
+     int F=0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'F' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4436382)